builder/struct.go

```diff
@@ -39,6 +39,7 @@ func (*Struct) Build(gen Generator, ctx *MethodContext, sourceID *xtype.JenID, s
 	usedSourceID := false
 	for i := 0; i < target.StructType.NumFields(); i++ {
 		targetField := target.StructType.Field(i)
+		targetTag := target.StructType.Tag(i)
 		delete(definedFields, targetField.Name())
```

Comment:
`matchTag` should support matchIgnoreCase.
```
package example

// goverter:converter
type Converter interface {
	// goverter:matchTag json
	// goverter:matchIgnoreCase
	Convertre(source Input) Output
}

type Input struct {
	A1 int `json:"a"`
}

type Output struct {
	B1 int `json:"A"`
}
```

Reply:
I don't think there is a clear rule how to name these, and having different casing there doesn't seem to be that rare:

- `userId` 1134 matches https://grep.app/search?q=json%3A%22userId&regexp=true&case=true
- `userID` 349 matches https://grep.app/search?q=json%3A%22userID&regexp=true&case=true
- `userid` 305 matches https://grep.app/search?q=json%3A%22userid&regexp=true&case=true
- `UserId` 75 matches https://grep.app/search?q=json%3A%22UserId&regexp=true&case=true